nodejs/week1/avg.js

```diff
@@ -0,0 +1,14 @@
+
+const array = process.argv.slice(2).map(num => Number(num));
+if (array.length === 0) {
+    console.log('No arguments are provided')
+}
```

Comment:
nice work with if and else statements just use less space and put them closer together